internals/maintenance_scripts.py

```diff
@@ -699,3 +700,106 @@ def get_template_data(self, **kwargs) -> str:
       ndb.delete_multi(keys_to_delete)
 
     return  (f'{counter} empty extension stages deleted.')
+
+
+LAST_MILESTONE_TO_YEAR: dict[int, int] = {
```

Comment:
Nit: Since this is always converted to a list of tuples, it could feasibly be that data type to begin with

Reply:
I just thought that the dictionary syntax is a little cleaner, with fewer parentheses.